doc/source/conf.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# BeamEnergy Device Class documentation build configuration file, created by
+# sphinx-quickstart on Tue Jul 25 13:20:18 2017.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = ['sphinx.ext.autodoc']
```

Comment:
In order to build documentation for Device Classes, it's quite possible that you'll need [`devicedoc`](https://github.com/vxgmichel/pytango-devicedoc) extension as well. I'm not sure how it is with Facades, though. Please, check if it builds simply like that.
Also, there's [`apidoc`](http://www.sphinx-doc.org/en/stable/man/sphinx-apidoc.html) which will generate your documentation files for you using `autodoc` directives.